forum/qa-theme/SnowFlat/js/selectLanguage.js

```diff
@@ -0,0 +1,72 @@
+document.addEventListener("DOMContentLoaded", function(event) {

```

Comment:
Arrow function możesz wstawić wszędzie, gdzie nie będziesz zmieniał **this**, a nie widzę abyś gdzieś tu z niego korzystał :)